docs/quickstart-packet.md

```diff
@@ -0,0 +1,269 @@
+INFO[14:19:49 EEST] Creating environment file…                    node=147.75.84.57
+INFO[14:19:49 EEST] Installing kubeadm…                           node=147.75.84.57 os=ubuntu
+INFO[14:20:36 EEST] Deploying configuration files…                node=147.75.80.241 os=ubuntu
+INFO[14:20:38 EEST] Deploying configuration files…                node=147.75.81.119 os=ubuntu
+INFO[14:20:40 EEST] Deploying configuration files…                node=147.75.84.57 os=ubuntu
+INFO[14:20:41 EEST] Generating kubeadm config file…
+INFO[14:20:42 EEST] Configuring certs and etcd on first controller…
+INFO[14:20:42 EEST] Ensuring Certificates…                        node=147.75.80.241
+INFO[14:20:54 EEST] Downloading PKI files…                        node=147.75.80.241
+INFO[14:20:56 EEST] Creating local backup…                        node=147.75.80.241
+INFO[14:20:56 EEST] Deploying PKI…
+INFO[14:20:56 EEST] Uploading files…                              node=147.75.81.119
+INFO[14:20:56 EEST] Uploading files…                              node=147.75.84.57
+INFO[14:21:01 EEST] Configuring certs and etcd on consecutive controller…
+INFO[14:21:01 EEST] Ensuring Certificates…                        node=147.75.81.119
+INFO[14:21:01 EEST] Ensuring Certificates…                        node=147.75.84.57
+INFO[14:21:11 EEST] Initializing Kubernetes on leader…
+INFO[14:21:11 EEST] Running kubeadm…                              node=147.75.80.241
+INFO[14:22:29 EEST] Joining controlplane node…
+INFO[14:22:29 EEST] Waiting 30s to ensure main control plane components are up…  node=147.75.84.57
+INFO[14:24:22 EEST] Waiting 30s to ensure main control plane components are up…  node=147.75.81.119
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.81.119
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.84.57
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.80.241
+INFO[14:26:22 EEST] Building Kubernetes clientset…
+INFO[14:26:26 EEST] Creating credentials secret…
+INFO[14:26:26 EEST] Ensure external CCM is up to date
+INFO[14:26:27 EEST] Patching coreDNS with uninitialized toleration…
+INFO[14:26:27 EEST] Applying canal CNI plugin…
+INFO[14:26:31 EEST] Installing machine-controller…
+INFO[14:26:35 EEST] Installing machine-controller webhooks…
+INFO[14:26:37 EEST] Waiting for machine-controller to come up…
+INFO[14:27:17 EEST] Creating worker machines…
+```
+
+KubeOne automatically downloads the Kubeconfig file for the cluster. It's named
+as **\<cluster_name>-kubeconfig**, where **\<cluster_name>** is the name from
+your configuration. You can use it with kubectl such as
+
+```bash
+kubectl --kubeconfig=<cluster_name>-kubeconfig
+```
+
+or export the `KUBECONFIG` variable environment variable:
+```bash
+export KUBECONFIG=$PWD/<cluster_name>-kubeconfig
+```
+
+## Scaling Worker Nodes
+
+As worker nodes are managed by MachineController, they can be scaled up and down
```

Comment:
By the way, is it called `Kuberentes scale API`? That `scale API` part sounds weird to me for some reason. Maybe it would be better to say `scale subresource`.

Reply:
changed to just `using Kubernetes API`